bundle/edu.gemini.p1backend/build.sbt

```diff
@@ -0,0 +1,26 @@
+import OcsKeys._
+
+// note: inter-project dependencies are declared in projects/OcsBundle.scala
+
+name := "edu.gemini.p1backend"
+
+// version set in ThisBuild
+
+libraryDependencies ++= Seq(
+  "org.typelevel" %% "cats-core" % "1.0.0-MF"
+  // The first one to use `cats-effects`. If not released yet, point to the
+  // git repo/branch in `dependsOn` in OcsBundle.scala.
+  // "org.http4s" %% "http4s-dsl" % "0.18.0-SNAPSHOT"
```

Comment:
The problem is that 0.16, 0.17 and 0.18 have all major changes in the exposed API. I'm positive 0.18 is going to be more definitive but the problem is not that much in http4s itself but in aligning the dependency versions in our code with http4s (cats, cats-effect, fs2).

By using the 0.18 dependencies we can have some certainty we won't have to rewrite the whole app in 6 months, I hope 🤞.

But even if we finish it very soon, which I don't think it'll happen because it's not that urgent, we won't put it in production until 2018B, minimum. Hopefully by then all the 3rd party APIs on which we depend will be settled.